administrator/language/de-DE/de-DE.plg_editors_codemirror.ini

```
@@ -15,9 +15,9 @@ PLG_CODEMIRROR_FIELD_AUTOCLOSETAGS_LABEL="Tag-Vervollständigung"
 ; The following two strings are deprecated and will be removed in J4
 PLG_CODEMIRROR_FIELD_AUTOFOCUS_DESC="Automatischer Fokus"
 PLG_CODEMIRROR_FIELD_AUTOFOCUS_LABEL="Automatischer Fokus"
-PLG_CODEMIRROR_FIELD_CODEFOLDING_DESC="<a href=\"https://de.wikipedia.org/wiki/Code-Faltung\" target=\"_blank\" rel=\"noopener noreferrer\">Code-Faltung</a> erlauben"
+PLG_CODEMIRROR_FIELD_CODEFOLDING_DESC="Code-Faltung erlauben.<br />Code-Faltung oder Code-Folding bezeichnet eine Funktion in Editoren, insbesondere in modernen integrierten Entwicklungsumgebungen, um logisch zusammengehörende Quelltextabschnitte wie Klassen oder Methoden in sogenannten Folds bzw. Falten zu gruppieren. Man kann die Abschnitte dann einfach ein- und ausblenden, als ob man ein Blatt Papier faltet, um bestimmte Zeilen auszublenden. Quelle: https://de.wikipedia.org/wiki/Code-Faltung"
```

hmm das ist mir persönlich doch etwas lang geworden,

```suggestion
PLG_CODEMIRROR_FIELD_CODEFOLDING_DESC="Code-Faltung erlauben.<br />Code-Faltung oder Code-Folding bezeichnet eine Funktion in Editoren, um logisch zusammengehörende Quelltextabschnitte in sogenannten Folds bzw. Falten zu gruppieren. Man kann die Abschnitte dann einfach ein- und ausblenden, als ob man ein Blatt Papier faltet, um bestimmte Zeilen auszublenden. Quelle: https://de.wikipedia.org/wiki/Code-Faltung"
```

---

Super